build/Dockerfile.odh-multi

```diff
@@ -0,0 +1,33 @@
+# Default to opendatahub-operator for the sake of odh autobuild
+ARG binary_name=opendatahub-operator
```

Comment:
I also have an idea about this -- build to a fixed name, and then add symlinks for kfctl and opendatahub-operator (and whatever else someone may want) so it can be invoked multiple ways and still autobuild.